docs/04-operation-guides/operations/02-install-kyma.md

```diff
@@ -68,6 +68,9 @@ A profile is defined globally for the whole Kyma installation. It's not possible
 
 If you install Kyma on a remote cluster, you can use the out-of-the box `kyma.example.com` domain. All you need to do is get your load balancer IP address and add the following line to the `hosts` file:
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```

Comment:
The warning feels a bit misplaced here since the rest of the document is not related to Jaeger. I think just removing it from the list of hostnames below should be fine. New users should not start using the deprecated Jaeger, but deploy their own one immediately.